packages/dataviz/src/LineChart/LineChart.component.js

```diff
@@ -0,0 +1,179 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import {
+	Legend,
+	LineChart,
+	Line,
+	CartesianGrid,
+	XAxis,
+	YAxis,
+	Tooltip,
+	ResponsiveContainer,
+} from 'recharts';
+import {
+	isSelectedOrHighlight,
+	getActiveDotR,
+	getDotR,
+	getStrokeOpacity,
+	getStrokeWidth,
+	union,
+} from './linechart-utils';
+
+import './LineChart.scss';
```

Comment:
here you globally override recharts linechart. I would prefer an integration using custom class, see below